mail_footer_notified_partner/models/mail_followers.py

```diff
@@ -9,9 +9,29 @@
 class MailNotification(models.Model):
     _inherit = 'res.partner'
 
+    @api.multi
+    def _notify_by_email(
+        self, message, force_send=False, send_after_commit=True,
+        user_signature=True
+    ):
+        # we need to save the complete list of partners because
+        # _message_notification_recipients builds recipients
+        # grouped by users groups. Thus get_additional_footer would get a
+        # partial list of recipients
+        ctx = dict(self.env.context)
+        ctx.update({"notified_partners": self})
+        return super(
+            MailNotification, self.with_context(ctx))._notify_by_email(
+                message, force_send, send_after_commit, user_signature)
+
     @api.model
     def _notify_send(self, body, subject, recipients, **mail_values):
-        body += self.get_additional_footer(recipients)
+        notified_partners = self.env.context.get('notified_partners')
```

Comment:
`.get('notified_partners', recipients)`

Reply:
@hbrunn actually ```notified_partners``` key could be present but empty. In that case, ```recipients``` should be used

Reply:
so + ` or recipients`

Reply:
Fixed